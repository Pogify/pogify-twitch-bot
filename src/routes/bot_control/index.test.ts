import botControlRouter from ".";

jest.autoMockOn();

test("test routes", async () => {
  expect(botControlRouter.stack).toMatchSnapshot();
});
