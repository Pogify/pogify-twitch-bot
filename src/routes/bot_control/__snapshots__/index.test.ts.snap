// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test routes 1`] = `
Array [
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "<anonymous>",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/\\?\\(\\?=\\\\/\\|\\$\\)/i,
    "route": undefined,
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "<anonymous>",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/\\?\\(\\?=\\\\/\\|\\$\\)/i,
    "route": undefined,
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/join\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "post": true,
      },
      "path": "/join",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "post",
          "name": "<anonymous>",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/part\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "post": true,
      },
      "path": "/part",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "post",
          "name": "<anonymous>",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/set\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "post": true,
      },
      "path": "/set",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "post",
          "name": "<anonymous>",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
  Layer {
    "handle": [Function],
    "keys": Array [],
    "name": "bound dispatch",
    "params": undefined,
    "path": undefined,
    "regexp": /\\^\\\\/current\\\\/\\?\\$/i,
    "route": Route {
      "methods": Object {
        "get": true,
      },
      "path": "/current",
      "stack": Array [
        Layer {
          "handle": [Function],
          "keys": Array [],
          "method": "get",
          "name": "<anonymous>",
          "params": undefined,
          "path": undefined,
          "regexp": /\\^\\\\/\\?\\$/i,
        },
      ],
    },
  },
]
`;
